build/scripts/minikube-tests/common.sh

```diff
@@ -45,19 +45,29 @@ initTemplates() {
   rm -rf ${CHECTL_TEMPLATES_BASE_DIR} && mkdir -p ${CHECTL_TEMPLATES_BASE_DIR} && chmod 777 ${CHECTL_TEMPLATES_BASE_DIR}
 
   PREVIOUS_OPERATOR_VERSION_CLONE_PATH=${OPERATOR_REPO}/tmp/${PREVIOUS_PACKAGE_VERSION}
-  git clone --depth 1 --branch ${PREVIOUS_PACKAGE_VERSION} https://github.com/eclipse-che/che-operator/ ${PREVIOUS_OPERATOR_VERSION_CLONE_PATH}
+  git clone --quiet --depth 1 --branch ${PREVIOUS_PACKAGE_VERSION} https://github.com/eclipse-che/che-operator/ ${PREVIOUS_OPERATOR_VERSION_CLONE_PATH}
 
   LAST_OPERATOR_VERSION_CLONE_PATH=${OPERATOR_REPO}/tmp/${LAST_PACKAGE_VERSION}
-  git clone --depth 1 --branch ${LAST_PACKAGE_VERSION} https://github.com/eclipse-che/che-operator/ ${LAST_OPERATOR_VERSION_CLONE_PATH}
+  git clone --quiet --depth 1 --branch ${LAST_PACKAGE_VERSION} https://github.com/eclipse-che/che-operator/ ${LAST_OPERATOR_VERSION_CLONE_PATH}
+
+  git clone --quiet https://github.com/devfile/devworkspace-operator ${OPERATOR_REPO}/tmp/dwo
+
+  local xFlag="+x"; [[ $- =~ x ]] && xFlag="-x"
+  set +x # suppress output
+  pushd ${OPERATOR_REPO}/tmp/dwo
+  DWO_STABLE_VERSION=$(git describe --tags $(git rev-list --tags) | sort --version-sort | tail -1)
```

Comment:
Executing this command in a freshly-cloned DWO repo has the error log

```
❯ git describe --tags $(git rev-list --tags) | sort --version-sort | tail -1     
fatal: No tags can describe '4b1a86952b6493f5541fa4a93aee4366d66d26fc'.
Try --always, or create some tags.
v0.16.0
```

I tried `git tag | sort -V` but the newest tag there is `v1.0.0-alpha1` from two years ago. However, I think another way to get the most recent tag is `git describe --tags $(git rev-list --tags --max-count=1)`

Reply:
Thank you, make sense to me.